lib/util/index.js

```diff
@@ -705,7 +705,7 @@ function isChromeScreenShareTrack(track) {
   // NOTE(mpatwardhan): Chrome creates screen share tracks with label like: "screen:69734272*"
   // we will check for label that starts with "screen:D" where D being a digit.
   const isChrome = util.guessBrowser() === 'chrome';
-  return isChrome && track.kind === 'video' && track.label && /^screen:[0-9]+/.test(track.label);
+  return isChrome && track.kind === 'video' && track.label && /^screen:[0-9]+/.test(track.label) && /^web-contents-media-stream:[0-9/]+/.test(track.label) && /^window:[0-9]+/.test(track.label);
```

Comment:
were you able to verify the change? Should that condition between tests be `||` ?

Reply:
@makarandp0 I was trying around with simpler signaling this morning and couldn't get the breakpoints to show up. Also oops, yeah the conditions should be `||`. Changing it now! 

I'll schedule something for the both of us!